package.json

```diff
@@ -186,6 +186,7 @@
     "lint:js": "eslint .",
     "lint:js:fix": "eslint --fix .",
     "lint:js:report": "eslint --output-file build/lint-js-report.json --format json .",
+    "lint:package-json": "npmPkgJsonLint .",
```

Comment:
Should this also be added to `lint-staged`?

Reply:
Great question.

`lint-staged` operates on individual staged files.

For that to work, `npmPkgJsonLint` would need to be able to consume individual files, but unfortunately it only seems to work on _directories_, e.g. `npmPkgJsonLint ./package.json` doesn't work. I get a `not a directory` error.

([source](https://npmpackagejsonlint.org/docs/en/cli))